identity-apps-core/apps/authentication-portal/src/main/webapp/oauth2_authz.jsp

```diff
@@ -62,6 +63,20 @@
     }
 
     boolean displayScopes = Boolean.parseBoolean(getServletContext().getInitParameter("displayScopes"));
+
+    final String authorizationDetailsParam = request.getParameter("authorization_details");
+    final Map<String, String> authorizationDetailsToBeDisplayed = new HashMap<>();
+    if (StringUtils.isNotBlank(authorizationDetailsParam)) {
+        final JSONArray authorizationDetails  = new JSONArray(authorizationDetailsParam);
```

Comment:
shall we wrap this in a try catch block and handle the invalid json scenario gracefully?

Reply:
Fixed with ef66c3344f1e03eb0923588042aa52f9f72ac807